web/angular-wallet/src/assets/particles.json

```diff
@@ -1,10 +1,11 @@
 {
+  "fps_limit": 30,
   "particles": {
     "number": {
-      "value": 80,
+      "value": 100,
       "density": {
-        "enable": true,
-        "value_area": 400
+        "enable": false,
```

Comment:
disabled the density, i.e. now we have 100 nodes per screen and not 80 nodes per 400x400 pixel